### PR TITLE
[SAH-19]: Configure Landing Page (Clean up)

### LINKED ISSUE #28: [SAH-19]: Configure Landing Page (Clean up)
## Details

* public folder missing, create one
* src folder missing, create one
* move app into src folder
* update next config file of landing page to enable the app server to run from the src directory
* create components folder
* add images & favicon
* update layout file by adding page title and font
* switch Inter font to Montserrat

### PR DESCRIPTION
**Pull request by mde3**
## Solved these issues
- [x] public folder missing, create one
- [x] src folder missing, create one
- [x] move app into src folder
- [x] update next config file of landing page to enable the app server to run from the src directory
- [x] create components folder
- [x] add images & favicon
- [x] update layout file by adding page title and font
- [x] switch Inter font to Montserrat

### CODE REVIEW

**Review by Maiz27 — APPROVED** [review]
LGTM

File: apps/website/next.config.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-module.exports = {
+const nextConfig = {
   reactStrictMode: true,
-  transpilePackages: ["ui"],
-};
+}
```
Comment (Emmanuel-Melon) [inline apps/website/next.config.js:3]: @Maiz27 @mde3 wondering why `transpilePackages: ["ui"]` was removed? Will the landing page follow a different design system?

**Review by Emmanuel-Melon — COMMENTED** [review]
Looks good!

File: apps/website/next.config.js (hunk 1 above)
Comment (mde3) [reply apps/website/next.config.js:3]: It's doesn't necessarily indicate a change in the design system. The removal was solely to address technical problems that emerged after relocating the application into the src directory.

Comment (Emmanuel-Melon) [reply apps/website/next.config.js:3]: Which technical problems?

Comment (mde3) [reply apps/website/next.config.js:3]: At the time when i moved the app folder into the src folder it brought this issue 

screencapture-localhost-3000

I can still add the transpilePackages: ["ui"] back, I had just figured the ui wouldn't be used on the landing page

**Pull request merged**